ocs_ci/ocs/mcg_workload.py

```diff
@@ -222,3 +222,20 @@ def _retrieve_job_state():
             log.debug(f"Logs from job pod {job_pod['metadata']['name']}: {pod_logs}")
 
         return False
+
+
+def get_pod_logs(job):
```

Comment:
Please update function `wait_for_active_pods` and replace lines 213-222 with call to this function.

Reply:
Yes, this can be changed.